src/views/Package/PackageDocs.tsx

```diff
@@ -48,171 +43,28 @@ const SubmoduleSelector: FunctionComponent = () => {
   ) : null;
 };
 
-const isApiPath = (path: string) => {
```

Comment:
I found this component file a bit overwhelming when I jumped into the task, so I extracted most of the business logic here into it's own module `useSectionItems`. This allows the `PackageDocs` component file to focus on presentation logic